models/user/userModel.js

```diff
@@ -0,0 +1,66 @@
+import mongoose, { Schema } from 'mongoose';
+import { addressSchema, orderItemSchema } from '../schemas';
+import { accountStatuses, roles } from '../../constants';
+import { phoneNumberPattern } from '../../helpers';
```

Comment:
Are you sure it will work without the extension /index.js?

Reply:
You're right, thank you for noticing.

Reply:
Fixed